src/AutoRest.CSharp.V3/Generation/Writers/XmlCodeWriterExtensions.cs

```diff
@@ -202,7 +202,8 @@ private static void ToDeserializeCall(this CodeWriter writer, XmlElementSerializ
             }
 
             var elementVariable = new CodeWriterDeclaration(serialization.Name.ToVariableName() + "Element");
-            using (writer.Scope($"if ({element}.Element({serialization.Name:L}) is {typeof(XElement)} {elementVariable:D})"))
+            writer.Line($"if ({element}.Element({serialization.Name:L}) is {typeof(XElement)} {elementVariable:D})");
+            using (writer.Scope())
```

Comment:
Why did this remove the using for the if-statement scope? It is the opposite change from the JSonCodeWriterExtensions.

Reply:
Because the variable inside the `if` is defined in the parent scope while the loop variable in `foreach` is defined in inner scope.